### PR TITLE
info recursively support fast mode

### PR DESCRIPTION
**Pull request by Hexilee**
Optimize `info -r` based on #3257

### CODE REVIEW

File: cmd/info.go [hunk 1]
```diff
@@ -108,11 +116,12 @@ func info(ctx *cli.Context) error {
...
 		}
 
-		wb := utils.NewBuffer(8 + 10)
+		wb := utils.NewBuffer(8 + 11)
 		wb.Put32(meta.InfoV2)
-		wb.Put32(10)
+		wb.Put32(11)
 		wb.Put64(inode)
 		wb.Put8(recursive)
+		wb.Put8(fast)
```
Comment (davies) [inline cmd/info.go:124]: client in old version will use it as raw

File: cmd/info.go [hunk 2]
```diff
@@ -59,6 +59,11 @@ $ juicefs info -i 100`,
 				Aliases: []string{"r"},
 				Usage:   "get summary of directories recursively (NOTE: it may take a long time for huge trees)",
 			},
+			&cli.BoolFlag{
+				Name:    "fast",
```
Comment (davies) [inline cmd/info.go:63]: Should we use the faster one as default? then we could add a option called `--accurate` or `--strict`

File: pkg/meta/utils.go [hunk 3]
```diff
@@ -369,3 +369,50 @@ func GetSummary(r Meta, ctx Context, inode Ino, summary *Summary, recursive bool
...
+	} else {
+		summary.Files++
+		summary.Length += attr.Length
+		summary.Size += uint64(align4K(attr.Length))
+	}
+	return 0
+}
+
+func fastGetSummary(r Meta, ctx Context, inode Ino, summary *Summary, recursive bool) syscall.Errno {
+	space, _, err := r.GetDirStat(ctx, inode)
```
Comment (davies) [inline pkg/meta/utils.go:389]: This should work in read-only mode

File: cmd/info.go (hunk 2 above)
Comment (Hexilee) [reply cmd/info.go:63]: Reasonable

File: pkg/meta/utils.go (hunk 3 above)
Comment (Hexilee) [reply pkg/meta/utils.go:389]: In #3282, the `GetDirStat` add a `trySync` parameter, it works in read-only mode if we disable `trySync`.

File: cmd/info.go (hunk 2 above)
Comment (Hexilee) [reply cmd/info.go]: Done

File: cmd/info.go (hunk 1 above)
Comment (Hexilee) [reply cmd/info.go]: Fixed

**Pull request merged**